### PR TITLE
test(hx-radio-control): test cases SURF2019

### PR DESCRIPTION
**Pull request by lalithkarikelli**
## Description

Test Case `<hx-radio-control>`

image

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF 2019

## Before you request a review for this PR:

- [x] Did you add component tests for any new code?
- [x] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [x] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/elements/hx-radio-control/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        it(`should NOT have a Shadow DOM`, async () => {
+            const component = /** @type {HXRadioControlElement} */ await fixture(template);
+            const shadow = component.shadowRoot;
+
+            expect(shadow).to.be.null;
+        });
+    });
+
+    describe('test Control Element', () => {
+        it('to test input[type="radio"]', async () => {
```
Comment (100stacks) [inline src/elements/hx-radio-control/index.spec.js:53]: Please modify to test the `controlElement` method:

```js
    get controlElement () {
        return this.querySelector('input[type="radio"]');
    }
```

As we discussed today, you can get the control element value using:

```js
const ctrl = fragment.controlElement;
```

File: src/elements/hx-radio-control/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            const elSelector = 'hx-radio-control > input#radBasicDemo';
+            const id = 'radBasicDemo';
+            const component = /** @type { HXRadioControlElement } */ await fixture(mockup);
```
Comment (100stacks) [inline src/elements/hx-radio-control/index.spec.js:56]: Please change variable `fragment`.

File: src/elements/hx-radio-control/index.spec.js (hunk 1 above)
Comment (lalithkarikelli) [reply src/elements/hx-radio-control/index.spec.js]: Changes have been done, please review.

File: src/elements/hx-radio-control/index.spec.js (hunk 2 above)
Comment (lalithkarikelli) [reply src/elements/hx-radio-control/index.spec.js]: Changes have been done, please review.

**Pull request merged**